script/openqa-clone-job

```diff
@@ -104,6 +104,10 @@ from JOBREF if it is a URL).
 Specifies the directory to store test assets (defaults to
 $OPENQA_SHAREDIR/factory).
 
+=item B<--skip-checks>
+
+Skips additional checks like maintenance update availability.
```

Comment:
**Introduce --skip-checks** where? The PR title is ambiguous.

I can see a change to `openqa-clone-job`?

Reply:
I added "in openqa-clone-job" in both the PR title and the corresponding commit message.